ecs_deploy/cli.py

```diff
@@ -30,6 +30,7 @@ def get_client(access_key_id, secret_access_key, region, profile):
 @click.option('-i', '--image', type=(str, str), multiple=True, help='Overwrites the image for a container: <container> <image>')
 @click.option('-c', '--command', type=(str, str), multiple=True, help='Overwrites the command in a container: <container> <command>')
 @click.option('-e', '--env', type=(str, str, str), multiple=True, help='Adds or changes an environment variable: <container> <name> <value>')
+@click.option('-s', '--secret', type=(str, str, str), multiple=True, help='Adds or changes a secret variable (Not available for Fargate): <container> <name> <value>')
```

Comment:
`<value>` might be a bit confusing here, as the user is not able to set the actual value here.
Maybe `<parameter name>` or something similar would be clearer.

Also "Adds or changes a secret variable" should be more specific I think, something like "Adds or changes a secret from the AWS Parameter Store"

Reply:
Yup, will fix.